cli/cli.go

```diff
@@ -246,13 +264,16 @@ func (cli *CLI) RunServer(ctx context.Context, args Args) error {
 		default:
 			return fmt.Errorf("invalid --index name %q", name)
 		}
-
-		db, err := sql.Open("sqlite3", path)
+		filename, err := cli.bzz.GetIndexFile(path, true)
+		if err != nil {
+			return err
+		}
+		db, err := sql.Open("sqlite3", filename)
 		if err != nil {
 			return err
 		}
 		defer db.Close()
-
+		defer cli.bzz.PutIndexFile(filename)
```

Comment:
The server only reads from the indexes, so is this necessary?

Reply:
On read only no. I'll remove.